frontend/src/stories/fixtures/activityLogList.json

```diff
@@ -5,12 +5,20 @@
     "timestamp": "2022-09-02T13:46:16.295020Z",
     "actorUser": 352,
     "effectedUser": 802,
+    "actorUserEmail": "faith.mofokeng@umuzi.org",
+    "effectedUserEmail": "faith.mofokeng@umuzi.org",
     "object1ContentTypeName": "git_real | pull request review",
     "object1Id": 53534,
     "object2ContentTypeName": "git_real | repository",
     "object2Id": 7250,
+    "object1Summary": {
+      "topicProgress": 16,
+      "card": 76,
+      "title": "Understanding the review column",
+      "flavourNames": []
+    },
+    "object_2Summary": null,
```

Comment:
you may not be using `object_2Summary` property but should it not be like `object1Summary`? there are a few instances of `object_2Summary` remaining, for the sake of keeping the data consistent, can you make sure they match